common/types/head.go

```diff
@@ -11,16 +13,20 @@ type Head[BLOCK_HASH Hashable] interface {
 	// ChainLength returns the length of the chain followed by recursively looking up parents
 	ChainLength() uint32
 
-	// EarliestInChain traverses through parents until it finds the earliest one
+	// EarliestHeadInChain traverses through parents until it finds the earliest one
 	EarliestHeadInChain() Head[BLOCK_HASH]
 
-	// Hash is the head's block hash
-	BlockHash() BLOCK_HASH
-
 	// Parent is the head's parent block
 	GetParent() Head[BLOCK_HASH]
 
+	// Hash is the head's block hash
+	BlockHash() BLOCK_HASH
+	GetParentHash() BLOCK_HASH
+
 	// HashAtHeight returns the hash of the block at the given height, if it is in the chain.
 	// If not in chain, returns the zero hash
 	HashAtHeight(blockNum int64) BLOCK_HASH
+
+	// ToInt returns the block height as a big.Int
+	ToInt() *big.Int
```

Comment:
The main reason why I included this method is because there are some parts of the headtracker debug log that uses `ToInt()`. I was thinking between this and just making use of `BlockNumber()` which returns int64. 
However, I ultimately chose to continue with the use of `*big.Int` for consistency and to avoid any conversion errors that could be important for debugging

Reply:
Longer term, I would like to have just this:
BlockNumber() big.Int

We shouldn't be using int64, when we know EVM uses big.Int.

Reply:
thanks for that note. Added a Jira ticket to note this change in the future.

Reply:
could we fix this now? it doesn't seem like a big fix...and means we don't need to add a backlog item

Reply:
that makes sense. I dont really like the naming of ToInt() as well. Thanks for the suggestion Aaron and Prashant!